packages/@azure/servicebus/data-plane/test/namespace.spec.ts

```diff
@@ -325,3 +328,44 @@ describe("Errors when send/receive to/from non existing Queue/Topic/Subscription
     should.equal(errorWasThrown, true, "Error thrown flag must be true");
   });
 });
+
+describe("Test createFromAadTokenCredentials", function(): void {
+  it("Create Namespace from AADTokenCredentials, send a message to the ServiceBus entity", async function(): Promise<
+    void
+  > {
+    const env = getEnvVars();
+    const tokenCreds = await msrestAzure.loginWithServicePrincipalSecret(
+      env.clientId,
+      env.secret,
+      env.tenantId,
+      { tokenAudience: aadServiceBusAudience }
+    );
+
+    if (!process.env.SERVICEBUS_END_POINT) {
+      throw new Error(
+        "Define SERVICEBUS_END_POINT in your environment before running integration tests."
```

Comment:
As discussed offline, lets not introduce a new variable and use regex to extract the host name from the already used `SERVICEBUS_CONNECTION_STRING`

Reply:
Yes, will do that.
Are we adding a sample on how to create a namespace using `createFromAadTokenCredentials` ?

Reply:
Good idea! Create a new issue for it

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/1252